package/MDAnalysis/analysis/base.py

```diff
@@ -184,6 +184,11 @@ class AnalysisBase(object):
     All results should be stored as attributes of the
     :class:`MDAnalysis.analysis.results.Results` container.
 
+    .. Note::
+       The instance attributes are populated during and on conclusion of
```

Comment:
```suggestion
       The instance attributes are created during and on conclusion of
```
? IMO it makes a bit clearer that they do not exist (as opposed to being there but not filled). Maybe we could also add something like "trying to access such attributes before `run()` will result in an `AttributeError`".